FHEM/14_SD_UT.pm

```diff
@@ -558,6 +562,9 @@ sub SD_UT_Parse($$) {
 		return "";
 	}
 
+	$iohash->{bitMSG} = $bitData;
```

Comment:
Hmm, in das IO Hash sollte man nicht direkt etwas schreiben finde ich.
Wozu wird das denn im IOHash benötigt?